cluster-ambari/src/main/java/com/sequenceiq/cloudbreak/ambari/status/AmbariClusterStatusService.java

```diff
@@ -75,4 +75,9 @@ public ClusterStatusResult getStatus(boolean blueprintPresent) {
     public Map<String, String> getHostStatusesRaw() {
         return ambariClient.getHostStatuses();
     }
+
+    @Override
+    public boolean isCMRunning() {
```

Comment:
`isCMRunning` in ambari class?

Reply:
Its CM as in Cluster Manager :trollface: